util/gs-cors.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+      "origin": ["*"],
+      "method": ["GET", "HEAD"],
+      "maxAgeSeconds": 86400
+    }
+]
```

Comment:
This is the JSON you need to set up the Google Storage bucket for every entity to be publicly accessible by default and accessible over CORS. I think it's important to keep this somewhere along with the code.